tools/project_manager/displayimage.php

```diff
@@ -57,7 +57,7 @@
 <input type="hidden" name="preload" value="<?php echo $preload; ?>">
 
 <?php echo _("Resize"); ?>:
-<input type="text" maxlength="3" name="percent" size="3" value="<?php echo $percent; ?>">%
+<input type="number" name="percent" min="1" max="100" value="<?php echo $percent; ?>">%
```

Comment:
Same as the previous one, might be a good idea to add `required`.  Does not handle `5e0` and `5.0` well but I guess that is not too important.